src/components/WalletButton.tsx

```diff
@@ -32,11 +32,19 @@ const WalletButtonWrapper = ({
   return (
     <Button
       variant="outlined"
-      sx={{padding: "10px 25px", backgroundColor: theme.palette.mode === "dark" ? "#1B1F1E" : "white",color: theme.palette.mode === "dark" ? "#2ED8A7" : "#121615"}}
+      sx={{
+        padding: "10px 25px",
+        backgroundColor: theme.palette.mode === "dark" ? "#1B1F1E" : "white",
+        color: theme.palette.mode === "dark" ? "#2ED8A7" : "#121615",
+      }}
       {...props}
     >
       {icon}
-      <Typography variant="body2" color={theme.palette.mode === "dark" ? "white" : "121615"} ml={2}>
+      <Typography
+        variant="body2"
+        color={theme.palette.mode === "dark" ? "white" : "121615"}
```

Comment:
`"121615"` is not a valid color.

Reply:
I'll leave this change to @msavir for #78 to avoid merge conflicts.

Reply:
@zihan-aptos could you remove this file from the PR then?